libs/akita-ng-entity-service/src/lib/ng-entity-service.loader.ts

```diff
@@ -1,14 +1,15 @@
-import { Subject } from 'rxjs';
-import { Injectable } from '@angular/core';
+import { ReplaySubject, Subject } from 'rxjs';
+import { inject, Injectable } from '@angular/core';
 import { filter, map } from 'rxjs/operators';
 import { isFunction } from '@datorama/akita';
 import { HttpMethod } from './ng-entity-service-notifier';
+import { NG_ENTITY_SERVICE_CONFIG } from '@datorama/akita-ng-entity-service';
 
 type Event = { method: HttpMethod; loading: boolean; storeName: string; entityId?: any };
 
 @Injectable({ providedIn: 'root' })
 export class NgEntityServiceLoader {
-  private dispatcher = new Subject<Event>();
+  private dispatcher = inject(NG_ENTITY_SERVICE_CONFIG)?.replayLoader ? new ReplaySubject<Event>() : new Subject<Event>();
```

Comment:
Is a function preferred here, or is this okay?

Reply:
I prefer a factory:

`{ connector: () => new ReplaySubjec(1) }`

Reply:
Sorry I'm familiar with factories but not 100% following in this context.

Something like this?

```typescript
const createDispatcherImpl = (config: NgEntityServiceGlobalConfig) => ({
  connector: () => config.replayLoader ? new ReplaySubject(1) : new Subject()
});

@Injectable({ providedIn: 'root' })
export class NgEntityServiceLoader {

  private dispatcher = createDispatcherImpl(inject(NG_ENTITY_SERVICE_CONFIG)).connector();
  // ...
}
```

Reply:
No. I mean that instead of the boolean `replayLoader`.

```ts
{ connector: () => new ReplaySubjec(1) }
```

Reply:
😅 Ah that makes sense, my bad! I replaced replayLoader boolean with connector subject factory and added that property to the documentation.